tests/kernel/smp/Kconfig

```diff
@@ -0,0 +1,12 @@
+# Copyright (c) 2020 Intel Corporation.
+# SPDX-License-Identifier: Apache-2.0
+
+config TRACE_SCHED_IPI
+	bool "Enable Test IPI"
+	default n
```

Comment:
Remove.   Boolean kconfig values are n by default already.

Reply:
Got it. thanks for your instruction.